script.js

```diff
@@ -1,4 +1,194 @@
+var generateOutcome = function () {
+  var randomInt = Math.floor(Math.random() * 3);
+  var map = { 0: "scissors", 1: "paper", 2: "stone" };
```

Comment:
nice! really like the use of objects instead of if statements to get the value associated with the number generated